src/modules/organization/queries.ts

```diff
@@ -8,3 +8,53 @@ export const ALL_INTERNAL_GROUP_POSITIONS = gql`
     }
   }
 `
+export const ALL_INTERNAL_GROUPS = gql`
+  query AllInternalGroups {
+    allInternalGroups {
+      id
+      name
+    }
+  }
+`
+
+export const ALL_INTEREST_GROUPS = gql`
+  query {
+    allInterestGroups {
+      id
+      name
+      type
+      groupImage
+      groupIcon
+    }
+  }
+`
+
```

Comment:
This should be a single custom query as mentioned above

Reply:
![image](https://user-images.githubusercontent.com/31571115/163443536-405425e1-60bd-4db9-bd41-413553f00e5e.png)

I'll try I guess